src/nextjs-front/pages/users/[id].tsx

```diff
@@ -167,36 +166,14 @@ const UserProfilePage: NextPageWithLayout = ({}) => {
   );
 
   const router = useRouter();
-  
+
+  /* Send DM to user */
   const handleMessage = () => {
-    setChatView('dm', 'direct message', { targetUsername: userId });
+    setChatView('dm', 'direct message', { targetUsername: userData.username });
```

Comment:
What happen if the usern changes his username ? Using the id to refer is the only data that does not change ? Why use the username ?

Reply:
This is the name displayed on the DM header (next to the status icon):

![Screenshot at 2022-03-23 10-24-09](https://user-images.githubusercontent.com/41596925/159666322-b89f9b18-6b1e-4309-a5d1-2d1fca3e0cff.png)